src/go/rpk/pkg/config/weak_test.go

```diff
@@ -7,6 +7,10 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+func makeIntPtr(i int) *int {
+	return &i
+}
+
```

Comment:
Can this be func local to `TestWeakBool`?

```go
intPtr := func( i int) *int { return &i }
```
since it is only used in that function, and only twice

Reply:
@twmb  Wdyt of making this a common util function? https://github.com/redpanda-data/redpanda/tree/dev/src/go/rpk/pkg/utils

Reply:
utils packages generally aren't recommended because they become a dumping ground of random things, which then makes each utility more and more difficult to discover. There is commonly a need to get a pointer to an int, but the solution is pretty small, and where to stuff the function is so ambiguous, that I've always done this local to my use case.

Reply:
Took the suggestion of making this a local func